mne/viz/epochs.py

```diff
@@ -442,6 +442,194 @@ def plot_epochs(epochs, epoch_idx=None, picks=None, scalings=None,
     return fig
 
 
+def plot_epochs_concat(epochs, picks=None, scalings=None, n_epochs=8,
```

Comment:
this number should determine the viewing window, so if < 8 are available the other "positions" should be blank